src/ui/settings/ViewSettingsMenu.qml

```diff
@@ -39,6 +39,13 @@ VPNFlickable {
             imageRightSource: "../resources/chevron.svg"
             pushView: "../settings/ViewLanguage.qml"
         }
+        ListElement {
+            //% "App Permissions" 
+            settingTitle: qsTrId("vpn.settings.appPermissions")
+            imageLeftSource: "../resources/settings/networkSettings.svg"
+            imageRightSource: "../resources/chevron.svg"
+            pushView: "../settings/ViewAppPermissions.qml"
```

Comment:
we should have a `VPN.perAppSupported` to control the visibility of this feature.